packages/tsx-editor/src/components/EditorPreview.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { Label } from 'office-ui-fabric-react';
+interface IEditorPreview {
```

Comment:
This needs to be exported and should be called IEditorPreviewProps (also please add a line before the declaration)